pkg/startup/startup.go

```diff
@@ -555,7 +555,11 @@ func configureIPsAndSubnets(node *api.Node) (bool, error) {
 
 	if ipv4Env == "none" && (ipv6Env == "" || ipv6Env == "none") && node.Spec.BGP.IPv4Address == "" && node.Spec.BGP.IPv6Address == "" {
 		log.Warn("No IP Addresses configured, and autodetection is not enabled")
```

Comment:
I think we should have two variants of this warning log.

One within the CALICO_NETWORKING_BACKEND branch, which is an error level log

One outside of it, that is Warning level and says something like this:

> No IP Addresses configured, and autodetection is not enabled. Some features may not work properly.